app/Http/Controllers/Store/CheckoutController.php

```diff
@@ -63,10 +63,10 @@ public function destroy()
         return ujs_redirect(route('store.products.index'));
     }
 
-    public function show()
+    public function show($id)
```

Comment:
a bit weird going to `Checkout#show` just to do `Checkout#store`. Sounds like the order is wrong. Maybe `Order#checkout` and `Order#process` instead? Or `update` instead of `store`. Though I think `/order/<number>/checkout` makes a bit more sense than `/checkout/<number>`; be it part of `Order` controller or pseudo-nested route.

Reply:
not changing any of the routes in this

Reply:
route cleaning is a separate thing